src/mixins/filing-template-mixin.ts

```diff
@@ -312,6 +324,38 @@ export default class FilingTemplateMixin extends Vue {
     // Will refactor above parse function into 1 generic filing parse function when we know more
   }
 
+  /**
+   * Parses a business snapshot into the store.
+   * @param businessSnapshot the business to be parsed
+   */
+  parseBusinessSnapshot (businessSnapshot: BusinessSnapshotIF): void {
+    // Set Business Information
+    this.setBusinessInformation(businessSnapshot.businessData.business)
+
+    // Set Name Translations
+    this.setNameTranslations(
+      businessSnapshot.businessAliases.aliases?.map(x => {
+        return {
+          value: x.alias,
+          oldValue: null,
+          action: null
+        }
```

Comment:
Nice, but would it make more sense to have oldValue and action nullable? (wondering not sayin')

Reply:
I don't follow entirely. What would the difference be here between defining them as null vs having them as nullable?

Reply:
If they were nullable, you wouldn't have to assign them to null here to avoid an object type error.

When I saw this, I wondered what those were used for, since you simply assigned them to null. I realize we are trying to use a structured type that matches what we ultimately send to the BE, but if the FE doesn't need them, maybe we don't need to assign them?

Anyway, no need to change this. I'm probably overthinking this 😁